openlibrary/templates/type/edition/view.html

```diff
@@ -461,11 +467,11 @@ <h4>$_("First Sentence")</h4>
 
             $if work.description and work.description != overview_desc:
               <h4>$_("Work Description")</h4>
-              <div class="work-description">
-                <div class="work-description-content restricted-view">
+              <div class="work-description read-more">
+                <div class="read-more__content" style="max-height:58px">
```

Comment:
Unfortunately not ; I was working very hard to avoid a content shift where the full content was visible and then not visible, so we need the height to be set early